docs/cis_self_assessment.md

```diff
@@ -777,22 +791,19 @@ Setting admission control policy to `NamespaceLifecycle` ensures that objects ca
 
 **Result:** Pass
 
-**Remediation:**
-Edit the API server pod specification file `/etc/kubernetes/manifests/kube-apiserver.yaml`
-on the master node and set the `--disable-admission-plugins` parameter to
-ensure it does not include `NamespaceLifecycle`.
-
 **Audit:**
+Run the below command on the master node.
 
-```
+```bash
 /bin/ps -ef | grep kube-apiserver | grep -v grep
 ```
 
-**Expected result**:
+Verify that the `--disable-admission-plugins` argument is set to a value that does not include `NamespaceLifecycle`.
 
-```
-'--disable-admission-plugins' is present OR '--disable-admission-plugins' is not present
-```
+**Remediation:**
+By default, RKE2 does not use this argument. If there's a desire to use this argument, follow the documentation and create ServiceAccount objects as per your environment. Then, edit the API server pod specification file `/var/lib/rancher/rke2/agent/pod-manifests/kube-apiserver.yaml`
```

Comment:
this refers to `ServiceAccount`. Bad copy pasta from 1.2.14 I think.

If you want, you can just simplify this one down to your statement about rke2 not using this argument and "No manual remediation needed."